care/emr/resources/scheduling/slot/spec.py

```
@@ -144,11 +144,18 @@ def perform_extra_serialization(cls, mapping, obj):
 
 class TokenBookingOTPReadSpec(TokenBookingBaseReadSpec):
     patient: PatientOTPReadSpec
+    associated_encounter: dict = {}
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)